app/models/tree.rb

```diff
@@ -3,10 +3,11 @@ class Tree
 
   attr_accessor :entries, :readme, :contribution_guide
 
-  def initialize(repository, sha, path = '/')
+  def initialize(repository, sha, path = '/', recursive = '0')
```

Comment:
Assignment Branch Condition size for initialize is too high. [28.6/15]<br>Perceived complexity for initialize is too high. [9/7]